Gems/Multiplayer/Code/Include/Multiplayer/ReplicationWindows/IReplicationWindow.h

```diff
@@ -50,6 +50,15 @@ namespace Multiplayer
         //! @param outNetworkRole output containing the network role of the requested entity if found
         virtual bool IsInWindow(const ConstNetworkEntityHandle& entityPtr, NetEntityRole& outNetworkRole) const = 0;
 
+        //! Adds an entity to the replication window's set
+        //! @param entity The entity to try adding
+        //! @return Whether the entity was able to be added
```

Comment:
I'm curious why adding might return false; adding an example would be nice. 
_"Each window is left to decide if it wishes to block an entity from being added; for example, ServerToClientWindow may wish to block entities outside of a client's awareness radius."_

Reply:
It's really dependent on the implementation of the specific ReplicationWindow.

Reply:
On first read I thought a false would be because of an error. Maybe because the comment "was able to be added", I put a lot of weight on ABLE.  It sounded like "we're going to try to add, but something might go wrong". Only after reading the implementation did I see "oh, not adding is desired; a good thing"